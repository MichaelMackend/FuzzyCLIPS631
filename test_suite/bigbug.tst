(set-strategy depth)
(unwatch all)
; bigbug4.bat test
(open "Results//bigbug.rsl" bigbug "w")
(dribble-on "Actual//bigbug.out")
(clear)
(release-mem)
(mem-used)
(load-facts bigbug.dat)
(clear)
(release-mem)
(mem-used)
(load bigbug.clp)
(reset)
(clear)
(release-mem)
(mem-used)
(dribble-off)
(load "compline.clp")
(printout bigbug "bigbug.bat differences are as follows:" crlf)
(compare-files "Expected//bigbug.out" "Actual//bigbug.out" bigbug)
; close result file
(close bigbug)
