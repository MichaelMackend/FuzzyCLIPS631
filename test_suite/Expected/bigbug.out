TRUE
FuzzyCLIPS> (clear)
FuzzyCLIPS> (release-mem)
4967 ;; A different number is OK
FuzzyCLIPS> (mem-used)
52109 ;; Reference mem-used number
FuzzyCLIPS> (load-facts bigbug.dat)
TRUE
FuzzyCLIPS> (clear)
FuzzyCLIPS> (release-mem)
379567 ;; A different number is OK
FuzzyCLIPS> (mem-used)
52109 ;; Should be the same as above
FuzzyCLIPS> (load bigbug.clp)
:
TRUE
FuzzyCLIPS> (reset)
FuzzyCLIPS> (clear)
FuzzyCLIPS> (release-mem)
379702 ;; A different number is OK
FuzzyCLIPS> (mem-used)
52109 ;; Should be the same as above
FuzzyCLIPS> (dribble-off)
